test/fw/ptl/utils/plugins/ptl_test_db.py

```diff
@@ -1612,18 +1613,33 @@ def __init__(self, dbtype, dbpath, dbaccess):
             raise PTLDbError(rc=1, rv=False, msg=_msg)
         elif not self.dbpath.endswith('.json'):
             self.dbpath = self.dbpath.rstrip('.db') + '.json'
+        self._data = {}
         self.jdata = {}
+        self.run_count = 1
         self.__cmd = [os.path.basename(sys.argv[0])]
         self.__cmd += sys.argv[1:]
         self.__cmd = ' '.join(self.__cmd)
         self.res_data = PTLJsonData(command=self.__cmd)
 
     def __write_test_data(self, data):
-        prev_data = copy.deepcopy(self.jdata)
-        self.jdata = self.res_data.get_json(data=data, prev_data=prev_data)
-        with open(self.dbpath, 'w') as fd:
-            json.dump(self.jdata, fd, indent=2)
-            fd.write("\n")
+
+        name = "Test_run_count:" + str(self.run_count)
+        if self.run_count != PtlTextTestRunner().run_count:
```

Comment:
`run_count` in PtlTextTestRunner is class variable, you don't have to instate instance of it to access run_count, you can access it like `PtlTextTestRunner.run_count`